dist/api_data.py

```diff
@@ -1304,8 +1304,8 @@ def __ge__(self, other):
         Config('file_metadata', '', r'''
             the file configuration extracted from the metadata of the export database'''),
         Config('metadata_file', '', r'''
-            File that contains all the relevant metadata information for the URI to import. The file
-            is generated by backup:export cursor.'''),
+            A text file that contains all the relevant metadata information for the URI to import.
+            The file is generated by backup:export cursor.'''),
```

Comment:
The initial character should not be capitalized and there should not be an ending period before the ending triple-quotes. Please also fix `compare_timestamp` in the same `import` section as it has the same capitalization issue.

Reply:
Done.